src/api/venue.ts

```diff
@@ -156,10 +156,8 @@ export const fetchDescendantVenues = async (
 };
 
 export const fetchRelatedVenues = async (
-  venueId: string
+  sovereignVenue: WithId<AnyVenue>
```

Comment:
This shouldn’t be changed like this

Reply:
https://github.com/sparkletown/sparkle/pull/1725#discussion_r664265175

Reply:
@0xdevalias @mike-lvov fixed in 98b0388